spyder/plugins/ipythonconsole/comms/kernelcomm.py

```diff
@@ -33,7 +33,7 @@ class KernelComm(CommBase, QObject):
 
     _sig_got_reply = Signal()
     _sig_channel_open = Signal()
-    sig_exception_occurred = Signal(str, bool)
+    sig_exception_occurred = Signal(dict)
```

Comment:
Please complete this signal using a docstring and include the interface of the `dict`.
```python
sig_exception_occurred = Signal(dict)
"""
.......
"""
```

Please also follow https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
No need to fix